public/components/directives/ui-filter-list.js

```diff
@@ -0,0 +1,147 @@
+import angular from 'angular';
+import _ from 'lodash';
+import getKeyCodes from 'keycode'
+ 
+export function uiFilterList($q, $window) {
```

Comment:
Presume this is a copy from Composer?

Reply:
(if so, and if trivial, the spec could come too 😁)

Reply:
(We don't run the tests in this project)

Reply:
haha, naw 😅

Reply:
Having a good look at this test, it uses a fair number of composer helpers.